js/src/Table.js

```diff
@@ -47,15 +50,15 @@ var TableView = widgets.DOMWidgetView.extend({
     redraw: function() {
         this.req.page_size = this.model.get('page_size');
         this.req.filters = this.model.get('filters');
-        firefly.showTable(this.el.id, this.req);
+        firefly.showTable(this.model.get('tbl_id'), this.req);
     },
 
     tableUpdated: function(action, state) {
-        if (action.payload.tbl_id === this.req.tbl_id) {
+        if (action.payload.tbl_id === this.model.get('tbl_id')) {
```

Comment:
We did not propagate tbl_id from the model to the request, hence tbl_id of the created table will mot match tbl_id of the model, and this if will always be false.

To pass tbl_id to the request, lines 26-37 should be
```
        var tbl_id = this.model.get('tbl_id');
        if (this.url_or_path.length === 0) {
            this.req = firefly.util.table.makeIrsaCatalogRequest('allwise_p3as_psd', 'WISE', 'allwise_p3as_psd',
                {
                    position: this.model.get('position'),
                    SearchMethod: 'Cone',
                    radius: this.model.get('radius')
                },
                {tbl_id}  // options
            );
        }
        else {
            this.req = firefly.util.table.makeFileRequest(this.model.get('title'), this.model.get('url_or_path'), null,
                     {
                         tbl_id,
                         page_size: this.model.get('page_size')
                     });
        }
```